src/test/java/build/buildfarm/common/BUILD

```diff
@@ -0,0 +1,10 @@
+java_test(
+    name = "DigestUtilTest",
+    srcs = ["DigestUtilTest.java"],
+    deps = [
+        "//src/main/java/build/buildfarm:common",
+        "//src/main/protobuf:build_buildfarm_v1test_buildfarm_java_proto",
+        "//3rdparty/jvm/com/google/truth",
```

Comment:
`deps` sorting is alpha (I think) standard from `buildifier`, with 3rdparty preceding src